src/tools/my_cpuid.c

```diff
@@ -152,7 +152,7 @@ const char* getCpuName()
         }
     }
     // failled, try to get architecture at least
-    f = popen("LC_ALL=C lscpu | grep -i \"architecture:\" | head -n 1 | sed -r 's/(architecture:)\\s{1,}//gi'", "r");
+    f = popen("uname -m", "r");
```

Comment:
historicaly, there was a hack using a fake `uname` to fool linux installer into thinking the arch was x86_64, and that was making this impossible to use. The hack is not usefull now, as box64 detect when a program to `uname -m` and redirect that to `box64 -m` to show "x86_64".

Reply:
Hmmm, but I'm still using x86 bash from time to time, so I don't like this change.

Reply:
what do you mean @ksco ?

Reply:
Sorry, I misunderstood.

Reply:
I think Box64 can refer to the source code of fastfetch to obtain CPU architecture

Reply:
> historicaly, there was a hack using a fake `uname` to fool linux installer into thinking the machine arch was x86_64, and that was making this impossible to use. The hack is not usefull now, as box64 detect when a program to `uname -m` and redirect that to `box64 -m` to show "x86_64".

use file /usr/bin/uname to check if uname is x86_64 or not

use /usr/bin/uname -m instead of uname -m